scripts/utils/trading_strategy_helpers.js

```diff
@@ -258,10 +255,16 @@ module.exports = function(web3 = web3, artifacts = artifacts) {
         const lowerLimit = lowestLimit * Math.pow(stepSizeAsMultiplier, bracketIndex)
         const upperLimit = lowerLimit * stepSizeAsMultiplier
 
-        const [upperSellAmount, upperBuyAmount] = calculateBuyAndSellAmountsFromPrice(upperLimit, targetToken)
+        const [upperSellAmount, upperBuyAmount] = calculateBuyAndSellAmountsFromPrice(
+          upperLimit,
+          18 + stableToken.decimals - targetToken.decimals
```

Comment:
This number could be negative, while it's supposed to be the number of decimals of an ERC20 token. In this case the function fails.